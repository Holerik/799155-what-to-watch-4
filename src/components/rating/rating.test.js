// footer.test.js
import React from 'react';
import renderer from 'react-test-renderer';
import Rating from './rating.jsx';


it(`<Rating /> should render correctly`, () => {
  const tree = renderer
    .create(<Rating
      index={0}
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
