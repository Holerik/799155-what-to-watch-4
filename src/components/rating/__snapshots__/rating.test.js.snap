// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Rating /> should render correctly 1`] = `
Array [
  <input
    className="rating__input"
    id="star-0"
    name="rating"
    type="radio"
    value="0"
  />,
  <label
    className="rating__label"
    htmlFor="star-0"
  >
    Rating 0
  </label>,
]
`;
